sigs/go/CHARTER.md

```diff
@@ -0,0 +1,48 @@
+# SIG Go
+
+## Objective
+
+The purpose of this special interest group is to foster and facilitate
+community collaboration around the TensorFlow Go bindings.  Community
+collaboration is expected to result in improvements to the usability and
+maintenance of the bindings.
+
+## Membership
+
+Anyone involved in or using or contributing to the TensorFlow Go bindings is
+welcome to join the group. To participate, please join the mailing list
+([go@tensorflow.org](mailto:go@tensorflow.org)).
+
+Archives of the mailing list will be publicly available.
+
+## Resources
+
+* Mailing list [SIG Go](https://groups.google.com/a/tensorflow.org/forum/#!forum/sig-go)
+* GitHub repository [tensorflow/go](https://github.com/tensorflow/go)
+* Vanity import path [tensorflow.org/go](https://tensorflow.org/go)
+
+## Quarterly Meetings
+
+SIG Go will hold quarterly meetings via public Google Video Conference for
```

Comment:
> would it be enough especially during the stage of SIG formation?

Good point.  Maybe meetings should be held monthly while the SIG is forming?  Lacking some other definition, I'm thinking that this should encompass completion of all SIG launch plan items in the SIG Request along with resolution to initial problems 2 and 3 (migrating bindings to new repository, restoring interoperability with `go get` tooling).

My hunch when drafting the charter was that, in the long-term, more frequent meetings (e.g., monthly) may be too much.

> also any plans for gitter? other SIGs use gitter, but it kind of depends on community preferences

Sure, I can add.  It seems that we could just deactivate the channel if it isn't used.  Possibly this is referenced generally in the charter ("other third-party communication tools") as to avoid driving a future revision to the charter if preferences change.

Reply:
it's more of ideas than suggestions :) I guess you can also discuss these topics during the first calls with the SIG